WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/EditOrderFormTests.swift

```diff
@@ -18,7 +18,7 @@ final class EditOrderFormTests: XCTestCase {
         // Then
         XCTAssertNotNil(try? sut.find(button: "Add Products"))
         XCTAssertNotNil(try? sut.find(button: "Add Custom Amount"))
-        XCTAssertNotNil(try? sut.find(button: "Add Customer"))
+        XCTAssertNotNil(try? sut.find(button: "Add Customer Details"))
```

Comment:
Note for reviewer: this test class was added (Aug 2024) while the feature was paused (May 2024) but the feature flag was enabled in debug mode. Expected strings were updated for the version when the feature flag is disabled.